script/client/spring/application.yml

```diff
@@ -110,3 +110,23 @@ seata:
       connect-timeout: 2000
       username: ""
       password: ""
+  saga:
+    enabled: false
+    state-machine:
+      application-id: ${spring.application.name:}
+      tx-service-group: ${seata.tx-service-group:}
+      table-prefix: seata_
+      enable-async: false
+      async-thread-pool:
+        core-pool-size: 1
+        max-pool-size: 20
+        keep-alive-time: 60
+      rm-report-success-enable: ${seata.client.rm.report-success-enable:false}
+      saga-branch-register-enable: ${seata.client.rm.saga-branch-register-enable:false}
```

Comment:
add config sagaJsonParser default fastjson
```
client {
  rm {
    sagaJsonParser = fastjson
  }
}
```
see: https://github.com/seata/seata/pull/2831

Reply:
I added it.